examples/react/w3console/src/app.tsx

```diff
@@ -201,7 +201,7 @@ export function Logo (): JSX.Element {
 
 export function App (): JSX.Element {
   return (
-    <W3APIProvider>
+    <W3APIProvider uploadsListPageSize={10}>
```

Comment:
moar! i think we'll add fake multipage pagination to our list component like certain other consoles do.
```suggestion
    <W3APIProvider uploadsListPageSize={60}>
```

Reply:
so like "fetch 3 pages of 20, show the first 20 as "page 1" and then show "page 2" and "page 3" as links and then "..." to imply more pages, and then fetch the next n when the user flips through the pages, kinda thing